maliput/tests/road_geometry_test.rs

```diff
@@ -97,28 +97,28 @@ fn backend_custom_command() {
     let road_geometry = road_network.road_geometry();
 
     let command = String::from("OpenScenarioLanePositionToMaliputRoadPosition,1,50,-1,0.");
-    let result = road_geometry.backend_custom_command(&command);
+    let result = road_geometry.backend_custom_command(&command).unwrap();
     assert_eq!(result, "1_0_-1,51.250000,0.000000,0.000000");
     let command = String::from("OpenScenarioRoadPositionToMaliputRoadPosition,1,50,0.");
-    let result = road_geometry.backend_custom_command(&command);
+    let result = road_geometry.backend_custom_command(&command).unwrap();
     assert_eq!(result, "1_0_-1,51.250000,1.000000,0.000000");
     let command = String::from("MaliputRoadPositionToOpenScenarioLanePosition,1_0_-1,51.250000,0.000000,0.000000");
-    let result = road_geometry.backend_custom_command(&command);
+    let result = road_geometry.backend_custom_command(&command).unwrap();
     assert_eq!(result, "1,50.000000,-1,0.000000");
     let command = String::from("MaliputRoadPositionToOpenScenarioRoadPosition,1_0_-1,51.250000,1.000000,0.000000");
-    let result = road_geometry.backend_custom_command(&command);
+    let result = road_geometry.backend_custom_command(&command).unwrap();
     assert_eq!(result, "1,50.000000,0.000000");
     let command = String::from("OpenScenarioRelativeRoadPositionToMaliputRoadPosition,1,0.,1.,50.,1.");
-    let result = road_geometry.backend_custom_command(&command);
+    let result = road_geometry.backend_custom_command(&command).unwrap();
     assert_eq!(result, "1_0_1,48.750000,1.000000,0.000000");
     let command = String::from("OpenScenarioRelativeLanePositionWithDsToMaliputRoadPosition,1,1,0.,-1,50.,-0.8");
-    let result = road_geometry.backend_custom_command(&command);
+    let result = road_geometry.backend_custom_command(&command).unwrap();
     assert_eq!(result, "1_0_-1,48.750000,-0.800000,0.000000");
     let command = String::from("OpenScenarioRelativeLanePositionWithDsLaneToMaliputRoadPosition,1,-1,0.,1,50.,0.8");
-    let result = road_geometry.backend_custom_command(&command);
+    let result = road_geometry.backend_custom_command(&command).unwrap();
     assert_eq!(result, "1_0_1,50.000000,0.800000,0.000000");
     let command = String::from("GetRoadOrientationAtOpenScenarioRoadPosition,1,50.,0.");
-    let result = road_geometry.backend_custom_command(&command);
+    let result = road_geometry.backend_custom_command(&command).unwrap();
```

Comment:
Can we add a test for when the backend_custom_command has a wrong command?

Reply:
Done.